app/lib/three_scale/models.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module Models
+    def all_models_with_a_table
+      return @all_models_with_a_table if defined?(@all_models_with_a_table)
+
+      Rails.autoloaders.main.eager_load_dir("#{Rails.root}/app/models")
```

Comment:
```suggestion
      Rails.autoloaders.main.eager_load_dir(Rails.root.join("app/models"))
```

Just something RuboCop suggests.

Reply:
What about this one? :grimacing: